src/mongo/util/text.cpp

```diff
@@ -42,6 +42,7 @@
 #endif
 
 #include "mongo/platform/basic.h"
+#include "mongo/base/parse_number.h"
```

Comment:
Please keep the includes in accordance with the [style guide](https://github.com/mongodb/mongo/wiki/Server-Code-Style#for-cpp-files).